Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2016 Smartsheet.com
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM java:8-jre-alpine
+EXPOSE 7090
+RUN mkdir -p /opt/mad/logs
+RUN mkdir -p /opt/mad/config/pipelines
+ADD target/appassembler /opt/mad
+ADD config /opt/mad/config
+WORKDIR /opt/mad
+ENV PARAMS --config file:/opt/mad/config/config.json
```

Comment:
As discussed this is incorrect. Main accepts a single argument which is a file path (not uri) to the configuration.